hcl2template/testdata/build/post-processor_onlyexcept.pkr.hcl

```
@@ -0,0 +1,17 @@
+
+// starts resources to provision them.
+build {
+    sources = [
+        "source.virtualbox-iso.ubuntu-1204"
+    ]
+
+    post-processor "amazon-import" {
+        only = ["virtualbox-iso.ubuntu-1204"]
+    }
+    post-processor "amazon-import" {
```

It is better if you use another post-processor here to make sure this one is being excluded and the other one added. 
```suggestion
    post-processor "manifest" {
```
You can add the mock for the `manifest` post-processor here: https://github.com/hashicorp/packer/blob/master/hcl2template/common_test.go#L32